packages/astro/src/@types/astro.ts

```diff
@@ -1412,34 +1412,51 @@ export interface AstroUserConfig {
 			/**
 			 * @docs
 			 * @name experimental.i18n.fallback
-			 * @type {Record<string, string[]>}
-			 * @version 3.*.*
+			 * @type {Record<string, string>}
+			 * @version 3.5.0
 			 * @description
 			 *
-			 * The fallback system of the locales. By default, the fallback system affect the **content only**, and it doesn't
-			 * do any redirects.
+			 * The content fallback strategy when navigating to pages in non-default languages that do not exist. (e.g. a translated page has not been created).
+			 *
+			 * This a plain object, where a key is a locale that hasn't traslated pages, and the corresponding value is the locale where you want to redirect the missing page to.
+			 *
+			 * #### Examples
+			 *
+			 * In this example, we configure Astro to redirect pages that belong to the `pt` locale to their `es` version, and pages that belong to the `fr` locale to their `en` version.
+			 *
+			 * ```js
+			 * export defualt defineConfig({
+			 * 	experimental: {
+			 * 		i18n: {
+			 * 			defuaultLocale: "en",
+			 * 			locales: ["en", "fr", "pt", "es"],
+			 * 			fallback: {
+			 * 				pt: "es",
+			 * 			  fr: "en"
+			 * 			}
+			 * 		}
+			 * 	}
+			 * })
+			 * ```
+			 */
+			fallback?: Record<string, string>;
+
+			/**
+			 * @docs
+			 * @name experimental.i18n.routingStrategy
+			 * @type {'prefix-always' | 'prefix-other-locales'}
+			 * @default {'prefix-other-locales'}
+			 * @version 3.5.0
+			 * @description
+			 *
+			 * Controls the routing strategy:
+			 *  - `prefix-other-locales`: This is the default value. When used, Astro will identify those routes that belong to the default locale and execute a redirect to a route stripped of that locale.
+			 *		Use `example.com/[lang]/content/` for every route.
+			 *  - `prefix-always`: When used, all URLs need to contains a locale prefix. Astro will return a 404 for any route that doesn't fullfill the requirements.
+			 * 		Use `example.com/[lang]/content/` for all non-default languages, but `example.com/content` for the default locale/.
```

Comment:
I'll leave this one for the other PR where we have a suggestion in progress!

Reply:
Yes, there aren't any changes here, only a different formatting